RELEASE.md

```diff
@@ -43,7 +47,9 @@ Request from the prep branch.
 This branch will be used to prepare all the necessary things to get ready for
 release.
 
-    git switch -c v$MAJOR.$MINOR.$PATCH-prep
+    git switch -c release-v$MAJOR.$MINOR.$PATCH-prep
+
+NOTE: Make sure this step is done from `v$MAJOR.$MINOR` branch, not `master`.
```

Comment:
nit: I think we could ensure this with

```bash
test "$(git rev-parse --abbrev-ref HEAD)" = "v$MAJOR.$MINOR" || git checkout "v$MAJOR.$MINOR"
```

prior to the `git switch` invocation.